File: mobile/src/main/java/uk/co/droidinactu/nanowrimo/db/ChatMessage.java
package uk.co.droidinactu.nanowrimo.db;

/**
 * Created by aspela on 11/04/17.
 */
public class ChatMessage extends AbstractDbObj {
    private static final String LOG_TAG = ChatMessage.class.getSimpleName() + ":";

    private String text;
    private String name;
    private String photoUrl;
    private String imageUrl;


    private String sender;
    private String recipient;
    private String datetime;
    private String msg;


    public ChatMessage() {
    }

    public ChatMessage(String text, String name, String photoUrl, String imageUrl) {
        this.text = text;
        this.name = name;
        this.photoUrl = photoUrl;
        this.imageUrl = imageUrl;
    }


    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPhotoUrl() {
        return photoUrl;
    }

    public void setPhotoUrl(String photoUrl) {
        this.photoUrl = photoUrl;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }
}

File: mobile/src/main/java/uk/co/droidinactu/nanowrimo/db/ChatMessage.kt
package uk.co.droidinactu.nanowrimo.db

/**
 * Created by aspela on 11/04/17.
 */
class ChatMessage : AbstractDbObj {

    var text: String? = null
    var name: String? = null
    var photoUrl: String? = null
    var imageUrl: String? = null


    private val sender: String? = null
    private val recipient: String? = null
    private val datetime: String? = null
    private val msg: String? = null


    constructor() {}

    constructor(text: String, name: String, photoUrl: String, imageUrl: String) {
        this.text = text
        this.name = name
        this.photoUrl = photoUrl
        this.imageUrl = imageUrl
    }

    companion object {
        private val LOG_TAG = ChatMessage::class.java.getSimpleName() + ":"
    }
}
